static/templates/gear_menu.hbs

```diff
@@ -8,6 +8,32 @@
             view is a Bootstrap nav tab, even though we don't show the tab
             anymore --}}
             <li class="invisible" style="display:none;" role="presentation"><a href="#message_feed_container" data-toggle="tab"></a></li>
+            <li class="org-info org-name">{{realm_name}}</li>
+            <li class="org-info org-url">{{realm_uri}}</li>
+            <li class="org-info org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Free</a>
+                {{else if is_plan_standard}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard</a>
+                {{else if is_plan_standard_sponsored_for_free}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard, sponsored for free</a>
+                {{else if is_self_hosted}}
+                    {{#tr}}
+                    Version {zulip_version}
+                    {{/tr}}
+                {{/if}}
+            </li>
+            {{#if (and is_plan_limited is_owner) }}
+            <li class="org-info">
+                <a href="/upgrade" target="_blank" rel="noopener noreferrer" role="menuitem">Upgrade to: Zulip Cloud Standard</a>
```

Comment:
What if we make "Zulip Cloud Standard" be a `plan_name` variable that is passed into these strings for translation? @alya for thoughts on whether that's the right way to do i18n, given that we don't translate the plan names anywhere else.

Reply:
Hm, we should perhaps mark "Upgrade to" for translation, but leave the plan names in English?

Reply:
That's what I suggested -- with the `{plan_name}` part in a variable, so that folks can still do language-appropriate word order for the part that isn't the plan name.